lib/confluence/vendor_specific.es6.js

```diff
@@ -0,0 +1,136 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+'use strict';
+
+require('./vendor_specific_data.es6.js');
+
+foam.CLASS({
+  package: 'org.chromium.apis.web',
+  name: 'VendorSpecific',
+  extends: 'org.chromium.apis.web.MetricComputer',
+  documentation: `VendorSpecific is a class that computes the Vendor Specific
+    API metrics from browserApiDAO. It shows the number of APIs
+    that only available in this browser.`,
+  requires: [
+    'org.chromium.apis.web.Browser',
+    'org.chromium.apis.web.WebInterface',
+    'org.chromium.apis.web.BrowserWebInterfaceJunction',
+    'org.chromium.apis.web.VendorSpecificData',
+    'foam.mlang.ExpressionsSingleton',
+    'foam.dao.ArraySink',
+    'foam.dao.EasyDAO',
+  ],
+  properties: [
+    {
+      name: 'vendorSpecificDAO',
+      documentation: `This is a DAO that contains vendor specific data.`,
+      final: true,
+      factory: function() {
+        return this.EasyDAO.create({
+          name: 'vendorSpecificDAO',
+          of: this.VendorSpecificData,
+          daoType: 'MDAO',
+        });
+      },
+    },
+  ],
+  methods: [
+    {
+      name: 'compute',
+      documentation: `Compute vendor specific API value for each browser in
+           browsers at the given date.`,
+      args: [
```

Comment:
Let's provide a default `compute()` implementation (even if it throws, saying that `compute()` is supposed to be abstract) in the base class. Put the documentation there, and omit it from all the sub-classes.